src/components/ShellBox/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './ShellBox.scss';
+
+interface Props {
+  textToCopy: string;
+  children: React.ReactNode;
+}
+
+const ShellBox = ({ children, textToCopy }: Props): JSX.Element => (
+  <pre className="shell-box">
+    <div className="shell-box-top">
+      <span>SHELL</span>
+      <button
+        type="button"
+        onClick={(event: React.MouseEvent<HTMLButtonElement>): void => {
+          event.preventDefault();
+          navigator.clipboard.writeText(textToCopy);
+        }}
+      >
+        copy
```

Comment:
Can we get some kind of feedback to let the user know that the text was copied?

Reply:
Added a copied text + done icon for 3 secs in f257538

![ErWYCXaKZc](https://user-images.githubusercontent.com/5341334/82853724-7fa50d00-9ed4-11ea-907b-d5748e669c88.gif)